tests/conftest.py

```diff
@@ -88,3 +90,10 @@ def tarsier(credentials):
             raise ValueError("Invalid OCR provider")
 
     yield Tarsier(ocr_service)
+
+
+@pytest_asyncio.fixture()
+async def browser_adapter_with_js(async_page: Page, tarsier: Tarsier) -> BrowserAdapter:
+    adapter = adapter_factory(async_page)
+    await tarsier._load_js(adapter)
+    return adapter
```

Comment:
This won't be re-used by any other tests. Let's add this to the same file as the other tests instead of this shared conftest file